api/include/opentelemetry/logs/event_id.h

```diff
@@ -18,14 +18,16 @@ namespace logs
 class EventId
 {
 public:
-  EventId(int64_t id, nostd::string_view name) noexcept
+  EventId(int64_t id, nostd::string_view name) noexcept : id_{id}
```

Comment:
not required to initialize `id_` inside constructor body if it is initialized using initializer list.

Reply:
Thanks for the catch.